command/harness/destroy.go

```diff
@@ -20,7 +27,29 @@ func HandleDestroy(ctx context.Context, r *VMCleanupRequest, s store.StageOwnerS
 	if r.StageRuntimeID == "" {
 		return ierrors.NewBadRequestError("mandatory field 'stage_runtime_id' in the request body is empty")
 	}
+	// We do retries on destroy in case a destroy call comes while an initialize call is still happening.
+	cnt := 0
+	b := createBackoff(destroyTimeout)
```

Comment:
What's the max retries that this loop will carry out in case vm is never found?

Reply:
```func createBackoff(maxElapsedTime time.Duration) *backoff.ExponentialBackOff {
	exp := backoff.NewExponentialBackOff()
	exp.MaxElapsedTime = maxElapsedTime
	return exp
}